src/schema/comments.ts

```diff
@@ -374,6 +526,21 @@ export const resolvers: IResolvers<any, Context> = {
         },
       );
     },
+    recommendedMentions: async (
+      _,
+      { postId, query, limit = 5 }: GQLMentionUserArgs,
+      ctx,
+    ): Promise<User[]> => {
+      const { con, userId } = ctx;
+      const ids = await (query
+        ? recommendUsersByQuery(con, userId, { query, limit })
+        : recommendUsersToMention(con, postId, userId, { limit }));
+
+      return con.getRepository(User).findByIds(
+        ids.filter((id) => id !== userId),
+        { order: { name: 'ASC' }, take: limit },
+      );
```

Comment:
This should be a `GraphORM` statement so it will return the exact fields needed by the GQL query

Reply:
Exactly. We need to use the `query` function like we do for feeds or anything else